package.json

```diff
@@ -59,5 +58,10 @@
       "npm run docs",
       "git add"
     ]
+  },
+  "husky": {
```

Comment:
How about a .huskyrc ?
See: https://github.com/balena-io-modules/balena-deploy-request/blob/master/.huskyrc

Reply:
I just ran the `husky-upgrade` tool and it changed it to this, I don't feel strongly about either format and just went with the least change/easiest upgrade path - ie keep this PR as close to a straight version bump as possible

Reply:
(if you do feel strongly I can do a separate PR for this)